export.py

```diff
@@ -1503,6 +1503,9 @@ def export_smoke(ri, ob):
     ri.Volume("box", rib_ob_bounds(ob.bound_box),
               smoke_res, params)
 
+def export_volume(ri,ob):
+    rm = ob.renderman
+    ri.Volume("box", rib_ob_bounds(ob.bound_box), [0, 0, 0])
```

Comment:
So technically there are other Volume implicit shapes, but the 99% case is using a box of course.